main.go

```diff
@@ -36,10 +39,13 @@ func Setup() *fiber.App {
 		log.Fatal(err)
 	}
 
-	publisherHandler := handlers.NewPublisher(gormDB)
-	softwareHandler := handlers.NewSoftware(gormDB)
-	statusHandler := handlers.NewStatus(gormDB)
-	logHandler := handlers.NewLog(gormDB)
+	go func() {
```

Comment:
**[suggestion]**
We could move outside the setup func to better clarify what is this

Reply:
That function needs a *gorm.DB, which is initialized in `Setup()`, maybe we can add a comment to highlight the function instead of moving it out.

Reply:
Yes, as you prefer!